templates/pages/admin/user.substitute.html.twig

```diff
@@ -0,0 +1,211 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2022 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{% set bg = '' %}
+{% if item.isDeleted() %}
+   {% set bg = 'asset-deleted' %}
+{% endif %}
+
+<div class="asset {{ bg }}">
+
+{# Form Header #}
+{% set target       = params['target'] ?? item.getFormURL() %}
+{% set canedit      = params['canedit'] ?? true %}
+{% set withtemplate = params['withtemplate'] ?? '' %}
+{% set rand         = random() %}
+{% set nametype     = params['formtitle'] ?? item.getTypeName(1) %}
+{% set friendlyname = params['friendlyname'] ?? item.getHeaderName() %}
+{% set no_id        = params['noid'] ?? false %}
+{% set id           = item.fields['id'] ?? -1 %}
+{% set formoptions  = params['formoptions'] ?? '' %}
+
+{% set entity_id = 0 %}
+{% set entity_name = '' %}
+{% if item.isEntityAssign() %}
+   {% if item.getType() == 'Entity' and item.fields['id'] == 0 %}
+      {% set entity_id = null %}
+   {% else %}
+      {% set entity_id = params['entities_id'] ?? item.getEntityID() ?? session('glpiactive_entity') %}
+   {% endif %}
+
+   {% if is_multi_entities_mode() %}
+      {% set entity_name = get_item_name('Entity', item.getEntityID()) %}
+   {% endif %}
+{% endif %}
```

Comment:
Many variables can be removed here. For instance:
- item has no `is_deleted` flag and `item.isDeleted()` is always false
- `withtemplate`, `noid`, `formtitle`, `friendlyname`, ... parameters are not passed, and some defined variables are probably useless

Reply:
solved with: cbc0d9afc2